aeron/clientconductor.go

```diff
@@ -399,7 +401,7 @@ func (cc *ClientConductor) AddSubscription(channel string, streamID int32) int64
 	return regID
 }
 
-func (cc *ClientConductor) FindSubscription(regID int64) *Subscription {
+func (cc *ClientConductor) FindSubscription(regID int64) (*Subscription, error) {
```

Comment:
Yes, this pattern is missing in a bunch of places where other code would throw exceptions, these APIs only call the generic error handler instead.